api/application/client.go

```diff
@@ -609,6 +612,7 @@ func (c *Client) ScaleApplication(in ScaleApplicationParams) (params.ScaleApplic
 		Applications: []params.ScaleApplicationParams{{
 			ApplicationTag: names.NewApplicationTag(in.ApplicationName).String(),
 			Scale:          in.Scale,
+			ScaleChange:    in.ScaleChange,
```

Comment:
We should validate scale args here, just like in.ApplicatioName above

Reply:
I'm not sure what validation makes sense here as remove-unit will pass this a negative number for the ScaleChange. Thoughts?

Reply:
No, I meant that we should use the same validation as on the server side, eg both can't be set